libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -125,18 +119,8 @@ async def _as_sync_iterator(*args: Any, **kwargs: Any) -> AsyncGenerator:
 class BaseChatModel(BaseLanguageModel[BaseMessage], ABC):
     """Base class for Chat models."""
 
-    cache: Optional[bool] = None
-    """Whether to cache the response."""
-    verbose: bool = Field(default_factory=_get_verbosity)
-    """Whether to print out response text."""
-    callbacks: Callbacks = Field(default=None, exclude=True)
-    """Callbacks to add to the run trace."""
     callback_manager: Optional[BaseCallbackManager] = Field(default=None, exclude=True)
```

Comment:
left this in the two places because deprecated and didn't want to add deprecated attr to baselanguagemodel